README.md

```diff
@@ -1,24 +1,41 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/kubernetes-node-exporter.svg?style=shield&circle-token=0a5aafcebabaed6f39a57293a96427f907674276)](https://circleci.com/gh/giantswarm/kubernetes-node-exporter)
+[![CircleCI](https://circleci.com/gh/giantswarm/node-exporter-app.svg?style=shield)](https://circleci.com/gh/giantswarm/node-exporter-app)
 
-# node-exporter Helm Chart
-Helm Chart for Node exporter in Guest Clusters.
+# node-exporter
 
-* Installs the [node-exporter service](https://github.com/prometheus/node_exporter).
+Helm Chart for Node Exporter running in Tenant Clusters.
+
+* Installs the [node-exporter service].
+
+## Deployment
+
+* Managed by [app-operator].
+* Production releases are stored in the [default-catalog].
+* WIP releases are stored in the [default-test-catalog].
 
 ## Installing the Chart
 
 To install the chart locally:
 
 ```bash
-$ git clone https://github.com/giantswarm/kubernetes-node-exporter.git
-$ cd kubernetes-node-exporter
-$ helm install helm/kubernetes-node-exporter-chart
+$ git clone https://github.com/giantswarm/node-exporter-app.git
+$ cd node-exporter-app
+$ helm install helm/node-exporter-app
 ```
 
 Provide a custom `values.yaml`:
 
 ```bash
-$ helm install kubernetes-node-exporter-chart -f values.yaml
+$ helm install node-exporter-app -f values.yaml
 ```
 
-Deployment to tenant clusters is handled by [chart-operator](https://github.com/giantswarm/chart-operator).
+ ## Release Process
+
+* Ensure CHANGELOG.md is up to date.
+* Create a new GitHub release which will push the tarball to the [default-catalog].
```

Comment:
Again does that mean pushing tag? I think the instruction here should be clearer. For me github release means that https://github.com/giantswarm/node-exporter-app/releases.

Reply:
Ack. I'll make the instructions more explicit. For node-exporter there is no release yet because we haven't tested the migration.

See https://github.com/giantswarm/metrics-server-app/releases for a migrated app.